.github/workflows/release.yml

```diff
@@ -190,16 +190,7 @@ jobs:
           fi
 
       - name: Install universal-sierra-compiler
-        shell: bash
-        run: |
-          if [[ ${{ matrix.target }} == *windows* ]]; then
-            curl -L -o udc.zip https://github.com/software-mansion/universal-sierra-compiler/releases/download/v2.0.0/universal-sierra-compiler-v2.0.0-x86_64-pc-windows-msvc.zip
-            unzip udc.zip
-            echo "UDC_PATH=$(pwd)/universal-sierra-compiler-v2.0.0-x86_64-pc-windows-msvc/bin/universal-sierra-compiler.exe" >> $GITHUB_ENV
-          else
-            curl -L https://raw.githubusercontent.com/software-mansion/universal-sierra-compiler/master/scripts/install.sh | sh
-            echo "UDC_PATH=${HOME}/.local/bin/universal-sierra-compiler" >> $GITHUB_ENV
-          fi
+        uses: software-mansion/setup-universal-sierra-compiler@v1
 
       - name: Smoke test
         shell: bash
```

Comment:
I can't comment on the lines below, but there we are using `UDC_PATH`, that is used in this Smoke tests job step - I think it should be removed, otherwise it may break smoke tests (UDC_PATH overwrites UNIVERSAL_SIERRA_COMPILER, and after this is merged I think it will set it to an empty string. This variable is being used by snforge/sncast if I recall correctly, so there will be no path to usc)

Reply:
Done!